packages/devtools_app/lib/src/screens/provider/instance_viewer/instance_providers.dart

```diff
@@ -257,7 +257,7 @@ Setter? _parseSetter({
       if (field.isFinal) return null;
       return mutate;
     },
-    orElse: () => throw FallThroughError(),
+    orElse: () => throw Exception(),
```

Comment:
Is there a message we could pass to exception that could help with context, if this ever gets hit?

Reply:
Will address in a follow up